mods/default/functions.lua

```diff
@@ -303,6 +304,16 @@ function default.register_fence(name, def)
 			connect_right = {{1/8,3/16,-1/16,1/2,5/16,1/16},
 				{1/8,-5/16,-1/16,1/2,-3/16,1/16}},
 		},
+		collision_box = {
+			type = "connected",
+			fixed = {-1/8, -1/2, -1/8, 1/8, 1/2 + fence_collision_extra, 1/8},
+			-- connect_top =
+			-- connect_bottom =
+			connect_front = {-1/8,-1/2,-1/2,1/8,1/2 + fence_collision_extra,-1/8},
+			connect_left = {-1/2,-1/2,-1/8,-1/8,1/2 + fence_collision_extra,1/8},
+			connect_back = {-1/8,-1/2,1/8,1/8,1/2 + fence_collision_extra,1/2},
+			connect_right = {1/8,-1/2,-1/8,1/2,1/2 + fence_collision_extra,1/8},
```

Comment:
- connect_right = {1/8,-1/2,-1/8,1/2,1/2 + fence_collision_extra,1/8},
+ connect_right = {1/8,-1/2,-1/8,1/2,1/2 + fence_collision_extra, 1/8}
(space and comma)

Reply:
What is wrong with this?
It follows the style that this file was using in keeping the last comma, even though it is not needed.

Reply:
Well, it wasn't bad to fix the style. Although this will not change anything. :)

Reply:
@MoNTE48
Fixed!